src/App.vue

```diff
@@ -156,7 +158,7 @@ function midiNoteOn(index: number, rawAttack?: number, channel?: number) {
     if (!midi.velocityOn) {
       rawRelease = 80
     }
-    if (whiteMode === 'simple') {
+    if (whiteMode === 'simple' || multichannel) {
```

Comment:
I guess this is on me for writing bad code in the first place. Enough churn. I'll fix it on `main`.